benches/benches/vm.rs

```diff
@@ -27,12 +27,15 @@ where
                 diff,
             } = &mut i;
             let code = OpcodeRepr::from_u8(instruction.op());
+            let original_db = vm.as_mut().database_mut().clone();
```

Comment:
Hmm, why do we want to work with database transaction here at all?

Reply:
To more accurately replicate the overhead the executor would actually have. Since executor will never use rocksdb directly without a view layer.

Reply:
But we can use `DatabaseTransaction` in the tests by default, then=)

Reply:
The issue is that we are trying to reuse the same vm instance across multiple benchmarks by using state diffs to undo the previous test run. This means we have to update the db on the vm instead of making a new vm each time.